### PR TITLE
Add question creation - Server

### PR DESCRIPTION
**Pull request by ReemCohen**
take a look on those changes

### CODE REVIEW

File: common/models/classes/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from "./Guid"
```
Comment (atlowChemi) [inline common/models/classes/index.ts]:
```suggestion
export * from "./Guid";

```

File: common/models/enums/QuestionType.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum QuestionType {
+    SingleChoice,
+    MultiChoice
+}
```
Comment (atlowChemi) [inline common/models/enums/QuestionType.ts]:
```suggestion
}

```

File: common/models/enums/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from "./QuestionType"
```
Comment (atlowChemi) [inline common/models/enums/index.ts]:
```suggestion
export * from "./QuestionType";

```

File: common/models/interfaces/Answer.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { guid } from "../classes";
+
+export interface Answer {
+    id?: guid;
+    content: string;
+    correct: boolean;
+}
```
Comment (atlowChemi) [inline common/models/interfaces/Answer.ts]:
```suggestion
}

```

File: common/models/interfaces/index.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export * from "./Answer"
+export * from "./Question"
```
Comment (atlowChemi) [inline common/models/interfaces/index.ts]:
```suggestion
export * from "./Answer";
export * from "./Question";

```

File: common/models/interfaces/Question.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+export interface Question {
+	id?: guid;
+	title: string;
+	additionalContent: string;
+	type: QuestionType;
+	answers: Answer[];
+	label: string;
+	alignment: "Horizontal" | "Vertical";
+}
```
Comment (atlowChemi) [inline common/models/interfaces/Question.ts:13]:
```suggestion
}

```

**Review by atlowChemi — CHANGES_REQUESTED** [review]
Good start, please ensure to add `;` at the end of lines, and to leave an empty line at the end of file...

**Review by ghost — APPROVED** [review]
Looking good

**Pull request merged**